src/formik/InitialValues.js

```diff
@@ -0,0 +1,12 @@
+const InitialValues = {
```

Comment:
[23.2](https://github.com/airbnb/javascript#naming--camelCase): Use camelCase when naming objects, functions, and instances.